runtimes/eden/src/pallets_system.rs

```diff
@@ -102,10 +103,11 @@ impl pallet_balances::Config for Runtime {
 	type ExistentialDeposit = ExistentialDeposit;
 	type AccountStore = frame_system::Pallet<Runtime>;
 	type WeightInfo = crate::weights::pallet_balances::WeightInfo<Runtime>;
-	type MaxHolds = ConstU32<0>;
+	type MaxHolds = ConstU32<2>;
 	type MaxFreezes = ConstU32<0>;
 	type FreezeIdentifier = ();
-	type RuntimeHoldReason = ();
+	type RuntimeHoldReason = crate::RuntimeHoldReason;
+	type RuntimeFreezeReason = ();
```

Comment:
Could `RuntimeFreezeReason` come from the same place as `RuntimeFreezeReason`?

Reply:
Looks like it should: https://github.com/paritytech/polkadot-sdk/commit/35eb133baab93d3f2f179df216b2cc175d7dcaf2